### PR TITLE
fix(security): do not allow to read files above

### PR DESCRIPTION
**Pull request by alexander-akait**
This PR contains a:

- [x] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [ ] **documentation update**
- [ ] **typo fix**
- [x] **metadata update**

### Motivation / Use-Case

security fix

### Breaking Changes

No

### Additional Info

No

### CODE REVIEW

File: src/utils/getFilenameFromUrl.js [hunk 1]
```diff
@@ -78,39 +116,51 @@ function getFilenameFromUrl(context, url) {
...
+        // eslint-disable-next-line no-param-reassign
+        extra.errorCode = 400;
+
+        return;
+      }
+
+      // ".." is malicious
+      if (UP_PATH_REGEXP.test(path.normalize(`./${pathname}`))) {
+        // eslint-disable-next-line no-param-reassign
+        extra.errorCode = 403;
```
Comment (megagreg72) [inline src/utils/getFilenameFromUrl.js:134]: This is the key change, right?

Comment (alexander-akait) [reply src/utils/getFilenameFromUrl.js:134]: yes

**Comment by jkeys089** [comment]
@alexander-akait is there any plan to backport this fix to v4? We have a project still on Nuxt 2 / Webpack 4.

**Comment by alexander-akait** [comment]
@jkeys089 Webpack 4 is deprecated, so if you want to do it, feel free to send a PR and I will make a release

**Pull request merged**